build.go

```diff
@@ -67,6 +72,10 @@ func runBuild(cmd *cobra.Command, targets []string) error {
 	var args []string
 	args = append(args, "build")
 	args = append(args, "--color=yes")
+	args = append(args, "--experimental_convenience_symlinks=ignore") // don't generate any convenience symlinks
```

Comment:
I assume this isn't necessary any more because we're symlinking to the build artifacts ourselves. Can you clarify in the comment that we're doing so so readers of the code aren't confused?

Reply:
Done.